src/extension.ts

```diff
@@ -222,6 +235,7 @@ async function createProvider(
   await registerProvider(extensionContext, provider, telemetryLogger);
   if (!commandDisposable) {
     commandDisposable = extensionApi.commands.registerCommand(MINIKUBE_MOVE_IMAGE_COMMAND, async image => {
+      if (!minikubeCli) throw new Error('minikube executable cannot be found');
```

Comment:
not related

Reply:
See https://github.com/podman-desktop/extension-minikube/pull/223#discussion_r1838047300